charts/osschallenge/README.md

```diff
@@ -56,6 +56,8 @@ This chart is maintained by [Adfinis](https://adfinis.com/?pk_campaign=github&pk
 | osschallenge.settings.emailPort | int | `587` | Port for the SMTP server |
 | osschallenge.settings.emailUser | string | `"example@example.com"` | Username for the SMTP server |
 | osschallenge.settings.siteUrl | string | `"https://oss-challenge.ch"` | URL for OSS-Challenge |
+| persistence.mountPath | string | `"/data"` | specifies where to mount the PV |
+| persistence.size | string | `"10Gi"` | specifies which size the PVC should request |
```

Comment:
I'm missing the documentation of `persistence.storageClassName` Have you used `helm-docs` for this?

Reply:
I used the pre-commit, which does use helm-docs

Reply:
can you recheck? I'm missing this: https://github.com/adfinis-sygroup/helm-charts/pull/174/files#diff-e7cb1aadd92c6e5a2393f8dddc7ceb280e160219d531b142cb850eaff7bfd7b8R113

Reply:
it's commented so it wont get doxed by helm-docs

Reply:
Would it be better to uncomment it, set it to nil and make a default-values.yml for the CI?

Reply:
Yes.

`storageClassName: ""`

Reply:
Ok, thanks. Did it like that.